backend/app/assets/stylesheets/spree/backend/vendor/index.scss

```diff
@@ -0,0 +1,11 @@
+@import "solidus_admin/select2";
+@import "solidus_admin/flatpickr/flatpickr";
+@import "prism";
+
+@import 'spree/backend/bootstrap_custom';
+@import 'solidus_admin/bootstrap/bootstrap';
+@import 'font-awesome';
+
+// These mixins will modify existing Bootstrap ones and need to be imported
+// after the Bootstrap has been processed.
+@import 'spree/backend/globals/mixins';
```

Comment:
Yes thanks for noticing, we should be good, all my checks were ok, but I'll re-run the reasoning below:

- _globals_ only contain variables and Sass mixins, and select2, flatpicker and prism are raw css files, not affected by any Sass specific variable or function
- `globals/mixins` are overrides of bootstrap mixins, so it's just needs to come after bootstrap and before our solidus_backend specific CSS
- font awesome is also a `.css.erb` file, that is unaffected by anything else

I blocked myself from cleaning up further because this is already EOL, just did the bare minimum to make it easy to define themes as variations of the current style sheet (e.g. via variable adjustment).